omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_bootstrap.validator.rb

```diff
@@ -0,0 +1,238 @@
+#
```

Comment:
Ugh. In an earlier iteration I had it this way, I then renamed it with _s , but then kept editing the one with .s.  I'll fix it. 

I don't think it's related @sdelano  - the sort would remain the same, but specific 'require_relative' fixed it.